ci/job.yml

```diff
@@ -0,0 +1,161 @@
+- job:
+    name: centos-container-pipeline-service-container-index
+    description: |
+        Managed by Jenkins Job Builder, do not edit manually!
+    node: container
+    properties:
+        - github:
+            url: https://github.com/CentOS/container-index
+    triggers:
+        - github-pull-request:
+            admin-list:
+              - kbsingh
+              - bamachrn
+              - mohammedzee1000
+              - dharmit
+              - navidshaikh
+            white-list:
+              - kbsingh
+              - bamachrn
+              - mohammedzee1000
+              - dharmit
+              - navidshaikh
+            cron: '* * * * *'
+            github-hooks: false
+            permit-all: true
```

Comment:
add 
```
             trigger-phrase: '#dotests'
```
also add other properties here well like in ci-functional ?  Like messages when its started, succeed, failed, error etc?

Reply:
@bamachrn : We need either `#dotests` or `ok to test` string right for triggering the CI via comment?

Reply:
by default it is `ok to test`

Reply:
I pointed out to streamline the trigger CI string in comment in both repositories.

Reply:
Added #dotests for index ci